CachetHQ/Cachet
# Allow editing incident updates

HI! :raising_hand_man: 

After reading #2618 I decided to give it a shot.

### What has been done:
- Created controller actions and views for editing an incident update;
- Added a query scope to find all incident updates by an incident;
- Placed a link in the frontend to the edit incident update page;
- Removed the update button from the incident's page;

### How to test:
- Checkout PR
- Make sure [your Cachet is up and running](https://docs.cachethq.io/docs/installing-cachet)
- Make sure you cachet has some incidents or updates, you can use the `php artisan cachet:seed` if required.
- Acknowledge the new incident update editing functionality.

### Notes:
- Clicking on an incident in the dashboard now routes you to it's updates page, I'm not sure if this is then intended flow you want users to take. Let me know if you have any other wishes. 
- In addition, I'm not sure about separating the controller or moving the view templates. If this setup doesn't work for you. Let me know.
- I'm would love to write some acceptance / functional tests to prove the functionality is working. But I could not find them in the project. Any suggestions on how you want this tested?
